web/portal/brand/src/entities/CallCsvScheduler/CallCsvScheduler.tsx

```diff
@@ -331,13 +331,23 @@ const properties: CallCsvSchedulerProperties = {
 
 type MarshallerType = typeof defaultMarshaller;
 const marshaller: MarshallerType = (values, properties) => {
-  values.company = values[values.companyType];
+  const isCompanyTypeAll = values.companyType === '__null__';
+  if (isCompanyTypeAll) {
+    values.company = null;
+  } else {
+    values.company = values[values.companyType];
+  }
```

Comment:
```suggestion
  if (isCompanyTypeAll) {
    values.company = null;
  } 
   
   values.company = values[values.companyType];
  
```

Reply:
But this doesn't have the same behavior. It's incorrect, and even if it weren't, it would be better to remove lines 335-337 altogether.